contrib/ceph/micro-osd.sh

```diff
@@ -0,0 +1,90 @@
+#
+#    Copyright (C) 2013,2014 Loic Dachary <loic@dachary.org>
```

Comment:
@noahdesu Thanks! It works fine after adding some config for the ssh client / hostname resolution in Circle :)